frontend/js/views/main.js

```diff
@@ -494,8 +577,10 @@ define(
                 goldenLayout.on("stateChanged", function (event) {
                     localStorage.setItem("layout-custom", JSON.stringify(goldenLayout.toConfig()));
                     localStorage.setItem("layout", "custom");
+                    annotationTool.playerAdapter.resetSize();
                 });
                 goldenLayout.init();
+                this.goldenlayout = goldenLayout; // TODO do not expose?
```

Comment:
Makes goldenLayout Object accessible from annotationTool object to use several api functions to improve usability; should be fine to add it